framework/auth/core.py

```diff
@@ -93,6 +94,15 @@ def validate_personal_site(value):
 def validate_social(value):
     validate_personal_site(value.get('personal'))
 
+
+def validate_email(item):
```

Comment:
Possibly related to the bug I reported: this email validator is probably incomplete. Could use something like https://github.com/marshmallow-code/marshmallow/blob/c781939daaee64bf55ffc73d893bcb7c67aeba0d/marshmallow/validate.py#L69-122, which is a simplification of Django's email validator: https://github.com/django/django/blob/master/django/core/validators.py#L137-L205